client/odysseus/message/style.scss

```diff
@@ -0,0 +1,88 @@
+.chatbox-message {
+	display: flex;
+	justify-content: space-between;
+	align-items: self-start;
+	margin-bottom: 8px;
+	max-width: 70%;
+	word-wrap: break-word;
+	padding: 12px;
+	border-radius: 4px;
+	font-size: $font-body-small;
+	line-height: 1.4;
+}
+
+.chatbox-message.user {
+	align-self: flex-end;
+	background-color: #dcf8c6;
+	color: #000;
+}
+
+.chatbox-message.wapuu {
+	align-self: flex-start;
+	background-color: #f1f0f0;
+	color: #000;
+	display: flex;
+	flex-direction: column;
+}
+
+.message-actions {
+	display: flex;
+	gap: 8px;
+	align-self: flex-end;
+	border: none;
+	background: transparent;
+	padding: 0;
+	margin: 0;
+	opacity: 1;
+	transform: scale(1);
+	visibility: visible;
+	transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, visibility 0.3s ease-in-out;
+}
+
+.message-action:focus {
+	outline: none;
+}
+
+.message-action {
+	cursor: pointer;
+	transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out, visibility 0.3s ease-in-out;
```

Comment:
I'm pretty sure we have a fade-in animation used somewhere in Calypso. Could we reuse it?